ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,23 @@
 <div class="container-fluid wu-frame">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
I have no idea why you think that we need styles other than the ones in Patternfly, but I will try this code again later.